python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -428,14 +428,6 @@ def core_loop(
                         thread_name_prefix="asset_daemon_worker",
                     )
                 )
-            num_submit_workers = self._settings.get("num_submit_workers")
```

Comment:
i took this out because it is currently unused and was a pain to make coexist with asyncio